src/design-system/Modal.tsx

```diff
@@ -8,6 +8,7 @@ const ModalContainer = styled.div``;
 
 export interface Props {
   modalTitle?: TitleProps["title"];
+  modalDescription?: string | React.ReactElement;
```

Comment:
Note, there is a new modal treatment as defined in [the mocks](https://www.figma.com/file/EGJS9njuKHWdFHUS4lrHA4/%F0%9F%9A%80Recidiviz-CJStatus?node-id=2733%3A130).  This optional `modalDescription` allows that new modal treatment to exist.